doc/source/dev/missing-bits.rst

```diff
@@ -0,0 +1,260 @@
+  used for the the 'versionadded' markup, [TBD: which is preferred?]:
+
+  * At the end of the description of the argument in the "Parameters" section
+  * In the "Notes" section.  In this case, the `versionadded` markup
+    wouldn't be used.  Instead, the new addition is noted with a plain
+    text comment.
+
+
+Citing wikipedia articles in the "References" section
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+It is acceptable to use wikipedia articles as references.
+When creating the citation for the reference, include the article title,
+the name "Wikipedia" (similar to how one gives a journal title), and the
+URL.
+
+    Yes::
+
+        .. [1] "Zeta Distribution", Wikipedia,
+               https://en.wikipedia.org/wiki/Zeta_distribution
+
+    No::
+
+        .. [1] https://en.wikipedia.org/wiki/Zeta_distribution    
+
+
+Use of ``np`` in the "Examples" section
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Do not include ``import numpy as np`` in the code given in the "Examples"
+section.  The NumPy Docstring standard says "The examples may assume that
+``import numpy as np`` is executed before the example code in numpy."  That
+statement makes the import *optional*; this guideline says explicitly
+that the import statement must not be included.
+
+
+Bulleted lists
+~~~~~~~~~~~~~~
+This is not so much a guideline as it is a reminder of the Sphinx markup
+for bulleted lists.  The incorrect use of indentation is common enough
+that it is worthwhile mentioning it here.
+
+When creating a bulleted list:
+
+* Don't end the preceding line with `::`.
+* Don't indent the bullets.
+* Include a blank line before and after the list.
+
+Some examples:
+
+    Yes::
+
+        Some text that precedes this interesting list:
+
+        * The first item in the list.
+        * The second item in the list.
+        * You get the idea.
+
+        Some text that follows the list.
+
+    No::
+
+        Some text that precedes this interesting list:
+
+          * The first item in the list.
+          * The second item in the list.
+          * You get the idea.
+
+        Some text that follows the list.
+
+    No::
+
+        Some text that precedes this interesting list:
+        * The first item in the list.
+        * The second item in the list.
+        * You get the idea.
+        Some text that follows the list.
+
+
+Last line of docstring is a blank line
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Include a blank line at the end of multiline docstrings.  (This refers
+to a final blank line *within* the docstring, not a blank line in the
+code that follows the docstring.)
+
+This is not part of [PEP-257](https://www.python.org/dev/peps/pep-0257/),
+and the examples in that PEP do not include the blank line.  It is
+also not explicitly stated in the 
+[NumPy docstring standard](https://numpydoc.readthedocs.io/en/latest/format.html),
+but the few complete examples shown there all include a blank line at
+the end.
```

Comment:
I've never really thought of blocking a PR for this either. Don't really mind it if you want to though.

Reply:
I do mind, this is a typical example of being too nitpicky, which is quite unpleasant for new contributors. Just push a fix before merging for something this small.

Reply:
@rgommers, that's fine.  I'm happy to remove this one, too.  Personally, I have no preference, and I don't think I would even bother pushing a "fix".  (I've probably written docstrings with and without a final blank line in the past.)

Reply:
I removed the guideline about including a blank line at the end of a docstring.  Everyone should understand that the implication of *not* having such a guideline is that we generally leave it up to the PR author to do as they please.